core/server/admin.js

```diff
@@ -105,7 +105,10 @@ app.use("/Admin", express.static(path.join(__dirname, '../clients/Admin')));
 app.use("/_common", express.static(path.join(__dirname, '../clients/_common')));
 
 app.get('/', function(req, res){
-    res.sendFile(path.join(__dirname, '../clients/Admin/index.html'));
+    if(req.query.s === secret || DEBUG === true)
```

Comment:
then, `=== secret` is weird. `req.query.s` is a string and `secret` an object, it will always be `false`